pycode/memilio-simulation/memilio/simulation/pybind_util.h

```diff
@@ -121,15 +193,18 @@ auto bind_Range(pybind11::module_& m, const std::string& class_name)
             pybind11::return_value_policy::reference_internal);
 
     //bindings for the range itself
-    pybind11::class_<Range>(m, class_name.c_str())
+    bind_class<PickleFlag::NoPickling, Range>(m, class_name.c_str())
```

Comment:
When would you choose which PickleFlag? Would it be possible to document somwhere a simple rule of thumb?